src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Tests/BaseTestServiceTest.cs

```diff
@@ -0,0 +1,107 @@
+﻿using AutoMapper;
+using DevAdventCalendarCompetition.Repository.Interfaces;
+using DevAdventCalendarCompetition.Repository.Models;
+using DevAdventCalendarCompetition.Services;
+using DevAdventCalendarCompetition.Services.Models;
+using DevAdventCalendarCompetition.Services.Profiles;
+using Moq;
+using System;
+using System.Collections.Generic;
```

Comment:
remove unused using

Reply:
2 tests fail because we don't have the logic for these tests